render/detailed_node.go

```diff
@@ -313,6 +313,18 @@ func containerOriginTable(nmd report.NodeMetadata, addHostTag bool) (Table, bool
 		rows = append(rows, Row{Key: "IP Address", ValueMajor: ip, ValueMinor: ""})
 	}
 
+	// Add labels in alphabetical order
+	labels := docker.ExtractContainerLabels(nmd)
+	labelKeys := make([]string, 0, len(labels))
+	for k := range labels {
+		labelKeys = append(labelKeys, k)
+	}
```

Comment:
You can do this more directly, e.g.

```go
labelKeys := make([]string, len(labels))
for i, k := range labels {
    labelKeys[i] = k
}
```

(This will go away with Tom's suggestion anyway, but just for the record...)